requirements.txt

```diff
@@ -1,7 +1,7 @@
 certifi==2020.4.5.2
 chardet==3.0.4
 idna==2.9
-numpy==1.18.5
+numpy==1.19.3
```

Comment:
Does this version work for you on Linux? Does requirements.txt allow splitting on platform as well?

Reply:
> For the pip package I think you can add a version range to `install_requires` in setup.py?

Oh yes good point - I forgot that in Python there are multiple places to specify dependencies...

> I.e. any version upto 1.19.3 on Windows and don't restrict on Linux?

What about macos, I wonder?

Reply:
> Does requirements.txt allow splitting on platform as well?

It seems like it does... at least when I run `pip3 install -r ...` it works fine and keeps me on `1.18.5`.

Reply:
> What about macos, I wonder?

I think macos would be "Darwin", but might be better to use `platform_system != "Windows"`

Reply:
OK - going with `... != "Windows"`.

Reply:
I think this is fixed...